### DISCUSSION TITLE
Relate a term to an entry?

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#8714] [General]
### Bug description

Hi

I've moved this over from [discussions](https://github.com/statamic/cms/discussions/5834)

### Is it possible to add a class to a taxonomy term based on its relationship to the current entry?

I'd like to highlight the related term by styling it with CSS.

I've got a sidebar listing taxonomy terms, these are mounted to a collection called `products`. 

The screenshot below shows an entry, the sidebar is a menu of taxonomy terms.

**Automated Machines** is the entry, and it's related to the term **Automation & Materials Handling**.

How would I test for that in antlers?
![](https://user-images.githubusercontent.com/72254741/163232230-191a4e2e-3b2c-4123-bdaa-b4f16db9b39a.png)
![](https://user-images.githubusercontent.com/72254741/163233788-bb496cf2-b59f-4b5d-b9c7-964a488766db.png)

The entry uses this view `resources/views/products/show.antlers.html`

This doesn't work

```
{{ if page:group_1 | raw | contains(slug) }}
  yup, the current page has this term
{{ /if }}
```

Switching back to `regex`, Erin suggested this
```
{{ if (page:group_1 raw="true" :contains="slug") }}
  yup, the current page has this term
{{ /if }}
```
Unfortunately this also doesn't work?

### How to reproduce

- Create an entry
- Assign a term to it
- On the entry view template output the all the taxonomy terms
- Try to highlight the term that's associated with the entry.

### Versions

```
Statamic 3.3.9 Pro
Laravel 9.5.1
PHP 8.0.8
simonhamp/statamic-heroicons 1.0.1
statamic/collaboration 0.4.0
Parser: runtime
```

### Installation

Fresh statamic/statamic site via CLI

### Antlers Parser

runtime (new)

**Comment by duncanmcclean** [comment]
What about something like this?

```{{ if slug == segment_2 }}```
